cdi-jdg/src/main/webapp/WEB-INF/beans.xml

```diff
@@ -24,4 +24,13 @@
 <beans xmlns="http://java.sun.com/xml/ns/javaee"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/beans_1_0.xsd">
+
+    <!-- Enable JCACHE interceptors -->
+    <interceptors>
+        <class>org.infinispan.jcache.annotation.InjectedCachePutInterceptor</class>
```

Comment:
//cc @wfink @mgencur 

This seems to be required for running this quickstart on EAP 6.4. Without it, EAP does not use JCache annotations.